### PR TITLE
Rename fv3gfs-util to pace-util

### PR DESCRIPTION
**Pull request by mcgibbon**
We've decided to rename the `fv3gfs` namespace to `pace`. This PR renames `fv3gfs.util` to `pace.util`.

Changes:
- fv3gfs-util and `fv3gfs.util` are changed to pace-util and `pace.util`
- added .dockerignore at the top level
- removed update_submodules_base make target from fv3core, since the submodule it updates no longer exists

### CODE REVIEW

File: fv3core/Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ CONTAINER_ENGINE ?=docker
 RUN_FLAGS ?=--rm
 TEST_ARGS ?=
 TEST_DATA_HOST ?=$(CWD)/test_data/$(EXPERIMENT)
-FV3UTIL_DIR=$(CWD)/external/fv3gfs-util
+FV3UTIL_DIR=$(CWD)/external/pace-util
```
Comment (elynnwu) [inline fv3core/Makefile:19]: [optional] Should we change this too?

**Review by elynnwu — APPROVED** [review]
Glad we chose the name `pace`, everything is more compact now!

**Comment by mcgibbon** [comment]
Tests are failing due to `fv3gfs-util` still being used in jenkins config, will update it today hopefully in a way that doesn't break PRs.

**Comment by mcgibbon** [comment]
launch jenkins

File: fv3core/Makefile (hunk 1 above)
Comment (mcgibbon) [reply fv3core/Makefile:19]: There are a bunch of references to fv3util too when doing `import pace.util as fv3util`. Not sure what we want to do with these.

**Pull request merged**